client/imports/ui/pages/course/course.ts

```diff
@@ -40,14 +43,33 @@
 // Export CourseView Class
   export default class CourseView extends MeteorComponent {
     courseId: string;
+    courseNumber: string = "";
+    isInstructor: boolean = false;
     user: Meteor.User;
     constructor(mdIconRegistry: MdIconRegistry, private route: ActivatedRoute) {
       super();
       // Create Icon Font
       mdIconRegistry.registerFontClassAlias('tux', 'tuxicon');
       mdIconRegistry.setDefaultFontSetClass('tuxicon');
+      this.subscribe('user-courses', () => {
+        this.autorun(() => {
+          this.courseNumber = Collections.courses.findOne({ _id: this.courseId }).course_number;
+        });
+      }, true);
     }
     ngOnInit() {
       this.courseId = (<any>(this.route.snapshot.params)).courseid;
+      if(typeof this.courseId !== "undefined") {
+        this.isInstructor = Roles.isInstructorFor(this.courseId);
+      }
+    }
+    isInstruct() {
```

Comment:
This is a needless function.  Just use the Roles.isInstructorFor function in the template itself.